**Biological network maps tutorial: from construction to data analysis and applications in research and clinics**
ICSB2016 satelite 

15 September, Barcelona

Venue: TBA

Time: 9.00-19.00

**Motivation**

Today's biology is largely data-driven, thanks to high-throughput technologies that allow investigating molecular and cellular aspects of life on large scales. Making biological sense out of the amount of produced data requires their interpretation in the context cellular and physiological processes. However the knowledge about molecular signalling mechanisms is dispersed in thousands of publications, precluding application of methods and algorithms developed in the field of bioinformatics and systems biology. There is a need in formalized compilation of biological knowledge in a computer-readable form, as bio-molecular networks.

To overcome the robustness of cell signalling network, disease treatment should be extended to a combination therapy approach. Integrating and analysing patient high-throughput data together with the information about biological signalling machinery may help deciphering molecular patterns specific to each patient and finding the best combinations of candidates for therapeutic targeting. Approaches in the field of systems medicine from the network biology perspective will be discussed.

**Learning objective**

1.   Expose the participants to several methodologies of network construction and data analysis in the context of networks.

2.   Provide the participants with number of optimized working scenarios that can be directly applied in their own research.

**Intended audience**

The tutorial targets a wide audience as computational systems biologists, molecular and cell biologists, clinicians, who are interested in update and learning of approaches in network biology and related analysis tools, including visualization and analysis of multi-level omics data in the context of biological networks. The presenters are computational systems biologists that will share their experience and suggest simple and proven scenarios on integration of the tools into basic and clinical research. No previous experience or computational background is required to attend the workshop. The tutorial will include hands-on sessions; the audience is advised to bring the laptop computers.
** **
**Important dates**

**18.07**       Early bird registration deadline ([Register here](http://www.icsb2016barcelona.org/registration/registration/))

**05.09**       Regular registration deadline ([Register here](http://www.icsb2016barcelona.org/registration/registration/))

**15.09         Biological network map tutorial**   

**16-21.09**    [ICSB](http://www.icsb2016barcelona.org/) main conference
** **
**Program**

**Session 1: Tools for network construction** 

**9.00-09.30**  Introduction lecture (Y. Matsuoka)

**9.30-10.30** Hands on session:  [CellDesigner](http://www.celldesigner.org/) ;  [PathVisio](http://www.pathvisio.org/) 

_*10.30-11.00 Coffee break*_

**Session 2: Pathway databases and networks resources** 

**11.00-11.30** Introduction lecture (R. Haw)

**11.30-13.30** Hands on session: [Reactome](http://www.reactome.org/) ; [Wiki Pathways](http://www.wikipathways.org/) ; [ACSN](https://acsn.curie.fr/) and [NaviCell](https://navicell.curie.fr/) ; [SIGNOR](https://en.wikipedia.org/wiki/Signor) 

*13.30-14.30 Lunch*

**Session 3: Tools for data visualization and analysis in the context of networks (part 1)** 

**14.30-15.00** Introduction lecture (T. Freeman)

**15.00-16.30** Hands on session: [BiNoM](https://binom.curie.fr/): [DEDAL](http://bioinfo-out.curie.fr/projects/dedal/) and [OCSANA](http://bioinfo-out.curie.fr/projects/ocsana/) ; [Cytoscape: Omic Analysis Collection](http://apps.cytoscape.org/apps/omicsanalysiscollection) ; [GSEA](http://www.broadinstitute.org/gsea), [ROMA](https://github.com/sysbio-curie/Roma) 

*16.30-17.00 Coffee break*

**Session 4: Tools for data visualization and analysis in the context of networks (part 2)**

**17.00-18.30** Hands on session: [MINERVA](http://r3lab.uni.lu/web/minerva-website/) ; [BioLayout Express3D](http://www.biolayout.org/); [Garuda platform](http://www.garuda-alliance.org/)

**18.30-19.00** Closing lecture: networks applications in clinics (E. Barillot ; M. Ostaszewski ; A. Mazein)

** **
**Downloadable materials**

Participants are encouraged to download didactic materials and install essential software before the tutorial.

** **
**Organizing committee**

**Presenters and tutors**

[Inna Kuperstein ](http://sysbio.curie.fr)(Institut Curie - INSERM - Mines ParisTech, Paris, France)

[Emmanuel Barillot](http://u900.curie.fr)  (Institut Curie - INSERM - Mines ParisTech, Paris, France)

[Robin Haw](http://www.reactome.org) (Ontario Institute for Cancer Research, Toronto, Canada)

[Tom Freeman](http://www.roslin.ed.ac.uk/tom-freeman/) (The Roslin Institute and University of Edinburgh, Edinburgh, UK)

[Yukiko Matsuoka](http://www.sbi.jp/members.htm) (Systems Biology Institute, Tokyo, Japan)

[Alexander Mazein](https://www.researchgate.net/profile/Alexander_Mazein) (European Institute for Systems Biology and Medicine, Lyon, France)

[Marek Ostaszewski ](https://www.researchgate.net/profile/Marek_Ostaszewski)(Luxembourg Centre for Systems Biomedicine, Universit√© du Luxembourg)

[Luana Licata](https://www.researchgate.net/profile/Luana_Licata/citations) (University of Rome Tor Vergata, Rome, Italy)

**Contributors**

[Andrei Zinovyev](http://www.ihes.fr/~zinovyev) (Institut Curie - INSERM - Mines ParisTech, Paris, France)

[Hiroaki Kitano](http://www.sbi.jp/) (Systems Biology Institute, Tokyo, Japan)

[Samik Ghosh](http://www.sbi.jp/members.htm) (Systems Biology Institute, Tokyo, Japan)

[Alexander Pico](http://profiles.ucsf.edu/alex.pico) (Gladstone Institutes, San Francisco, USA)

[Livia Perfetto](https://www.researchgate.net/profile/Livia_Perfetto2) (University of Rome Tor Vergata, Rome, Italy)

** **

**Tools description**

[CellDesigner](http://www.celldesigner.org/)

CellDesigner is a structured diagram editor for drawing gene-regulatory and biochemical networks. Networks are drawn based on the process diagram, with graphical notation system, and are stored using the Systems Biology Markup Language (SBML), a standard for representing models of biochemical and gene-regulatory networks. Networks are able to link with simulation and other analysis packages through Systems Biology Workbench (SBW).

[PathVisio](http://www.pathvisio.org/) 

PathVisio is a free open-source pathway analysis and drawing software. It allows you to draw, edit and analyse biological pathways. PathVisio provides a basic set of features for pathway drawing, analysis and visualization. Additional features are available as plugins. You can find a list of all available plugins in the plugin repository. Plugins are available for pathway building, pathway analysis, import/export functionality, data visualization or data integration.

[Reactome](http://www.reactome.org/)
Reactome is a free, open-source, curated and peer reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education. 

[Wiki Pathways](http://www.wikipathways.org/)

WikiPathways is an open, collaborative platform dedicated to the curation of biological pathways. WikiPathways thus presents a new model for pathway databases that enhances and complements ongoing efforts, such as KEGG, Reactome and Pathway Commons. The open, public approach of WikiPathways allows for broader participation by the entire community, ranging from students to senior experts in each field. This approach also shifts the bulk of peer review, editorial curation, and maintenance to the community.

[ACSN](https://acsn.curie.fr/)  

ACSN is a pathway database and a web-based environment that contains  interconnected cancer-related signalling network maps. molecular mechanisms involved in cancer are represented in the form of comprehensive interconnected maps created using systems biology standards, making these maps amenable for computational analysis. ACSN aims to collect, organize and graphically represent non-disease molecular mechanisms lying behind well defined cancer hallmarks (Hanahan and Weinberg). ACSN concentrates on exhaustive representation of molecular mechanisms that are frequently dysregulated in cancer such as cell cycle, DNA repair, cell death, survival, immune response signalling, telomere maintenance and others.

[NaviCell](https://navicell.curie.fr/) 

NaviCell is a web-based environment for exploiting large maps of molecular interactions, created in CellDesigner. It is characterized by a combination of three essential features: (1) efficient map browsing based on Google Maps; (2) semantic zooming for viewing different levels of details or of abstraction of the map and (3) integrated web-based blog for collecting community feedback. NaviCell Web Service allows mapping multilevel high-throughput data onto biological network maps, browsable with NaviCell.

[SIGNOR](https://en.wikipedia.org/wiki/Signor) 

The SIGnaling Network Open Resource, SIGNOR is a database of causal relationships between biological entities. It was was developed to support such a strategy by providing a scaffold of prior experimental evidence of causal relationships between biological entities. The information captured in SIGNOR can be represented as a signed directed graph illustrating the activation/inactivation relationships between signalling entities. Each entry is associated to the post-translational modifications that cause the activation/inactivation of the target proteins.

[BiNoM](https://binom.curie.fr/)

[Cytoscape: Omic Analysis Collection](http://apps.cytoscape.org/apps/omicsanalysiscollection)

[GSEA](http://www.broadinstitute.org/gsea)

[ROMA](https://github.com/sysbio-curie/Roma) 

[MINERVA](http://r3lab.uni.lu/web/minerva-website/) 

[BioLayout Express3D](http://www.biolayout.org/)

[Garuda platform](http://www.garuda-alliance.org/)

** **

**Contacts**

inna.kuperstein@curie.fr

emmanuel.barillot@curie.fr

** **
